docs/core/docker/build-container.md

```diff
@@ -146,46 +135,48 @@ From the working folder, enter the *app* folder with the example source code and
 dotnet publish -c Release
 ```
 
-This command compiles your app to the *publish* folder. The path to the *publish* folder from the working folder should be `.\app\bin\Release\netcoreapp2.2\publish\`
+This command compiles your app to the *publish* folder. The path to the *publish* folder from the working folder should be `.\app\bin\Release\netcoreapp3.1\publish\`
 
 Get a directory listing of the publish folder to verify that the *myapp.dll* was created. From the *app* folder, run one of the following commands:
 
 ```console
-> dir bin\Release\netcoreapp2.2\publish
- Directory of C:\docker-working\app\bin\Release\netcoreapp2.2\publish
-
-04/05/2019  11:00 AM    <DIR>          .
-04/05/2019  11:00 AM    <DIR>          ..
-04/05/2019  11:00 AM               447 myapp.deps.json
-04/05/2019  11:00 AM             4,608 myapp.dll
-04/05/2019  11:00 AM               448 myapp.pdb
-04/05/2019  11:00 AM               154 myapp.runtimeconfig.json
+> dir bin\Release\netcoreapp3.1\publish
+
+    Directory:  C:\docker-working\app\bin\Release\netcoreapp3.1\publish
+
+Mode                LastWriteTime     Length Name
+----                -------------     ------ ----
+-a---          1/9/2020  11:41 AM      407   myapp.deps.json
+-a---          1/9/2020  11:41 AM     4.50KB myapp.dll
+-a---          1/9/2020  11:41 AM   166.00KB myapp.exe
+-a---          1/9/2020  11:41 AM      672   myapp.pdb
+-a---          1/9/2020  11:41 AM      154   myapp.runtimeconfig.json
 ```
 
 ```bash
-me@DESKTOP:/docker-working/app$ ls bin/Release/netcoreapp2.2/publish
+me@DESKTOP:/docker-working/app$ ls bin/Release/netcoreapp3.1/publish
 myapp.deps.json  myapp.dll  myapp.pdb  myapp.runtimeconfig.json
 ```
 
 ## Create the Dockerfile
 
-The *Dockerfile* file is used by the `docker build` command to create a container image. This file is a plaintext file named *Dockerfile* that does not have an extension.
+The *Dockerfile* file is used by the `docker build` command to create a container image. This file is a plaintext file named *Dockerfile* that doesn't have an extension.
 
-In your terminal, navigate up a directory to the working folder you created at the start. Create a file named *Dockerfile* in your working folder and open it in a text editor. Add the following command as the first line of the file:
+In your terminal, navigate up a directory to the working folder you created at the start. Create a file named *Dockerfile* in your working folder and open it in a text editor. Depending on the type of application you're going to containerize, choose the ASP.NET Core runtime or the .NET Core runtime. When in doubt, choose the ASP.NET Core runtime, which includes the .NET Core runtime.
```

Comment:
Not clear why you're being told to choose when the intro points to a different tutorial for ASP.NET.  "the type of app your're going to containerize" is the one you just created, which isn't ASP.NET.

Reply:
I'm going to try and clarify this. One point on this is to help educate that there is in fact two different runtimes. The product team has specified that on the linux install instructions (for all the different distros) to promote installing the ASP.NET Core runtime over the .NET Core runtime as it has the most compatibility. So I'm kind of doing the same here.